packages/components/src/template-registry.ts

```diff
@@ -62,6 +62,9 @@ import type HdsSideNavListLinkComponent from './components/hds/side-nav/list/lin
 import type HdsSideNavListTitleComponent from './components/hds/side-nav/list/title';
 import type HdsStepperStepIndicatorComponent from './components/hds/stepper/step/indicator';
 import type HdsStepperTaskIndicatorComponent from './components/hds/stepper/task/indicator';
+import type HdsTabsComponent from './components/hds/tabs';
+import type HdsTabsPanelComponent from './components/hds/tabs/panel';
+import type HdsTabsTabComponent from './components/hds/tabs/tab';
```

Comment:
nit
```suggestion
import type HdsTabsTabComponent from './components/hds/tabs/tab';
import type HdsTabsPanelComponent from './components/hds/tabs/panel';
```

Reply:
Should these not be in alphabetical order?

Reply:
I don't know anymore 😀